integration-tests/README.md

```diff
@@ -94,6 +98,8 @@ You can define these environment variables in localconfig.properties file or via
 
 Check the [localconfig-example.properties](localconfig-example.properties) file for an example of how to use it.
 
+Also you need to create a localconfig.yaml file containing external system credentials and URLs for TNB framework. Check the [localconfig-example.yaml](localconfig-example.yaml) file for an example of how to use it.
```

Comment:
@Karel,
I believe we can remove slack related environment variables from `Remote system access configuration` as we dont need these variables to run tests locally  and its configured in `localconfig.yaml`

Reply:
Good catch, thanks, fixed